src/components/app/FooterLinks.css

```diff
@@ -0,0 +1,28 @@
+.appFooterLinks {
+  position: absolute;
+  bottom: 0;
+  right: 0;
+  border-top: 1px solid var(--grey-40);
+  border-left: 1px solid var(--grey-40);
+  padding: 0px 3px;
+  background-color: var(--grey-20);
```

Comment:
nit: when there are many properties in a rule (let's say: 6 or more), I have the habit of grouping alike properties together, with an empty line between groups. In that case I try to loosely follow https://github.com/necolas/idiomatic-css#declaration-order: 1/ positioning  2/ layout 3/ other

I find this easier to read :)